web/styles/popovers.css

```diff
@@ -833,14 +833,47 @@ ul {
         margin: 0;
     }
 
-    /* Override the default border-radius to properly align
-       the button corners with `stream_header_colorblock`. */
-    .dropdown-toggle {
-        border-radius: 1px 4px 4px 1px !important;
+    .move-topic-dropdown {
+        display: flex;
+        justify-content: flex-start;
```

Comment:
`flex-start` is the default, so no need to set it here unless you're setting it to something else, elsewhere.

Reply:
Yeah! fixed.